frontend/src/App.test.tsx

```diff
@@ -155,39 +139,48 @@ describe("App", () => {
     expect(instance.connectionManager.disconnect).toHaveBeenCalled()
   })
 
-  it("reloads when streamlit server version changes", () => {
-    const props = getProps()
-    const wrapper = shallow(<App {...props} />)
-
-    // A HACK to mock `window.location.reload`.
-    // NOTE: The mocking must be done after mounting,
-    // but before `handleMessage` is called.
-    const { location } = window
-    // @ts-ignore
-    delete window.location
-    // @ts-ignore
-    window.location = { reload: jest.fn() }
-
-    const fwMessage = new ForwardMsg()
-    fwMessage.newSession = {
-      config: {},
-      initialize: {
-        environmentInfo: {
-          streamlitVersion: "svv",
+  describe("streamlit server version changes", () => {
+    let prevWindowLocation: Location
+    beforeEach(() => (prevWindowLocation = window.location))
+    afterEach(() => (window.location = prevWindowLocation))
+
+    it("triggers page reload", () => {
+      const props = getProps()
+      const wrapper = shallow(<App {...props} />)
```

Comment:
we should refactor this test to react jest and remove enzyme

Reply:
Actually, idk if this test can be refactored? Maybe we can think about it more though?